example/push_transaction.dart

```diff
@@ -4,6 +4,9 @@ main() {
   EOSClient client = EOSClient('http://127.0.0.1:8888', 'v1',
       privateKeys: ["5J9b3xMkbvcT6gYv2EpQ8FD4ZBjgypuNKwE1jxkd7Wd1DYzhk88"]);
 
+  //EOSClient client = EOSClient('http://127.0.0.1:8888', 'v1');
+  //client.privateKeys = ["5J9b3xMkbvcT6gYv2EpQ8FD4ZBjgypuNKwE1jxkd7Wd1DYzhk88"];
```

Comment:
any reason to comment out this?

Reply:
Commented that out just to illustrate an example of how you could add private keys if not through the constructor.

Reply:
It's kind of unnecessary though, should we remove it?

Reply:
You can keep it but make it like an example. e.g. remove line 7, keep line 8 but mentioning `you can set the privateKeys after client was constructed, for example, xxxx`